### ISSUE TITLE
Allow option to show the selected game PIC in main menu

### ISSUE THREAD

**Issue by kenshin088** [ONElua/ONEMenu-for-PSVita#23]
Very rough mockup of how it would look to load the selected game's PIC as the background for the main menu. I think it would look really nice though I don't know if it will take up a lot of memory or not. Maybe a nice fade transition for the PIC when going from game to game.

![2017-11-23-033046](https://user-images.githubusercontent.com/33103311/33164198-c9be073a-cfff-11e7-9e3d-f9c3946fbe81.jpg)

**Comment by szpieg-z-krainy-deszczowcow** [comment]
Bad idea because Vita resolution is 960x544 and game pics are 840x500. Background image would be blurred.

The same bad idea is scalling squared icons. ;) Default are 128x128 but ONEMenu never displaying them natively, only smaller or larger so they are also blured.

**Comment by kenshin088** [comment]
Easy fix. Make the top header and footer black bars. It will look fine. Even if thr images are zoomed in, if you  out on a little filter like to make it opaque then it will hardly be noticable. It sould still look nice imo.

**Comment by ONElua** [comment]
szpieg-z-krainy-deszczowcow:
1menu to be port of the PSP version...the number of icons to show, coordinates .were already planned that way...
to modify everything according to the size of each icon it would take me more time ...
since the psx games are 80x80, those of psp are 144 * 80...

kenshin088 :
making that transition it slows down from one game to another game...

PD: imagine 50imgs * 960 * 544 * 4 = 99.609375MB...

sorry my bad engrish...

**Comment by szpieg-z-krainy-deszczowcow** [comment]
Don't get me wrong. I fully understand You, and appreciate the work and effort You have put into this project. Yet still I will be constructive critical in some parts of the UX which I don't like or think it's broken. I wrote this because I found in those: "..." added on the end of sentences as indicate of feeling been irritated and/or offended. ^^ So don't worry. It's not my intension.

 I have PSTV and find ONEMenu as a nice replacement for bubbles which isn't the best UI for "non touchable" devices. That's the main reason why I care of the future of this project.

My proposition is displaying selected menu items in their native resolutions (128x128) and non selected scalled down to be smaller. I never bought anything emulated on PSV so I don't know how meta data of PSX and PSP content looks like on Live Area. I know that PSX icons for PS3 & PSP XMB are to small and PSP icons for PSP are wide screen proportions which they are also to small to displaying natively on PSV/PSTV. So maybe especially for them, preparing scalled up nearest neighbour menu objects (but keeping original aspect ratio)? And if we talking about different displaying content types, maybe for PSV games & homebrew displaying circles (most of the themes are prepared to be displayed in such way making the rest of the area just black or white which doesn't looks good in ONEMenu)? Or completely dropping down this idea and focusing on custom icons/pic for cover flow menu type?

Transitions between read backgrounds pics will slow down only if You trying read them at the same time as user put "cursor". It will be ok, if they be read only after i.e ~1s or 2s after no controls pressed which means user stop browsing. It;s kind of dirty solution but it works for weak devices.

PS: Don't worry about English, be proud of Your native language. English is just a tool for us to communicate like in the past Latin and in the future, who knows, maybe Chinese. ;) My English is also far to be perfect.
